docs/dynamic-plugins/faq.md

```diff
@@ -0,0 +1,154 @@
+# Frequently Asked Questions
+
+## What is Dynamic plugin derived package?
+
+This is a special JavaScript package that is derived from an original plugin package source code.
+You find more information about process of creating derived packages in the [Export Derived Dynamic Plugin Package](export-derived-package.md) document.
+The derived plugins shouldn't be pushed into the [public npm registry](https://www.npmjs.com), but it can be published to a private or internal npm registry.
+You can find more information about publishing dynamic plugins in the [Packaging Dynamic Plugins](packaging-dynamic-plugins.md) document.
+
+## How to configure up a dynamic plugin project to publish to a npm registry?
+
+See [Creating JavaScript package](packaging-dynamic-plugins.md#creating-javascript-package) in [Packaging Dynamic Plugins](packaging-dynamic-plugins.md) document.
+
+## How to configure RHDH to use private or internal npm registry?
+
+See [Using a custom NPM registry](installing-plugins.md#using-a-custom-npm-registry) in [Installing Plugins](installing-plugins.md) document.
+
+## How to install community Backstage plugin to the RHDH?
```

Comment:
That's nice. But at some point, when it's ready and container-based exports are in place, we should probably point to how to leverage the overlay repo.

Reply:
yes, agreed. But even after we have the overlay repo, this will be still useful for plugins that are not in community-plugins repo. Maybe I should change the title to make it clear that this process should work for any plugin